org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/parser/AbstractCommentParser.java

```diff
@@ -349,9 +348,32 @@ protected boolean commentParse() {
 						lineHasStar = false;
 						// Fix bug 51650
 						this.textStart = -1;
+						// Only reset textStart if not in an unclosed markdown inline tag
+						if (!(this.markdown && this.inlineTagStarted)) {
+							this.textStart = -1;
+						}
 						this.markdownHelper.resetAtLineEnd();
-						if (this.inlineTagStarted && this.markdown) {
-							isTagElementClose = true;
+						if (this.markdown && this.inlineTagStarted && this.index < this.javadocEnd - 2) {
+							// Special handling for markdown comments with unclosed inline tags
+							if (this.source[this.index] == '\r') this.index++;
+							if (this.source[this.index] == '\n') this.index++;
+
+							// Skip exact '///' sequence if present
+							if (this.index + 2 < this.javadocEnd &&
```

Comment:
Not sure if expecting a /// right after newline. If I introduce any whitespace in the next line before ///, then the new tests fail. For e.g. 
```
				/// {@link #getValue()
				  /// value}} }dsfsdf {@link #getValue()
				  /// value1 } test
				public class IncorrectTagElement {}
```

Reply:
Handled this scenario.